docs/dev/upgrade-opentelemetry.md

```diff
@@ -4,18 +4,22 @@ When the [opentelemetry-collector](https://github.com/open-telemetry/opentelemet
 
 ## Update All Dependencies
 
-There is a `make` target that will update all `opentelemetry-collector-contrib` dependencies, which will include all indirect `opentelemetry-collector` dependencies:
+There is a `make` target to update all OpenTelemetry dependencies and regenerate necessary metadata. To run the update:
+```
+make update-opentelemetry
+```
+
+Read on for details of the actual details of the target, or skip ahead to [Test](#test) to continue the process.
```

Comment:
"details of the actual details"? I think you meant `s/for details of/for/`…
```suggestion
Read on for the actual details of the target, or skip ahead to [Test](#test) to continue the process.
```

Reply:
Deleted this line when I changed it to a zip section